apps/web/app/(app)/(survey-editor)/environments/[environmentId]/surveys/[surveyId]/edit/components/AddQuestionButton.tsx

```
@@ -46,22 +47,33 @@ export const AddQuestionButton = ({ addQuestion, product, isCxMode }: AddQuestio
-              addQuestion({
-                ...universalQuestionPresets,
-                ...getQuestionDefaults(questionType.id, product),
-                id: createId(),
-                type: questionType.id,
-              });
-              setOpen(false);
-            }}>
-            <questionType.icon className="text-brand-dark -ml-0.5 mr-2 h-4 w-4" aria-hidden="true" />
-            {questionType.label}
-          </button>
+          <TooltipProvider delayDuration={50}>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)